src/components/Icon/Icon.jsx

```diff
@@ -31,7 +31,7 @@ export const Icon = ({ name, color, size }) => (
 );
 
 Icon.propTypes = {
-  name: PropTypes.oneOf(['github', 'play', 'pause', 'restart']),
+  name: PropTypes.oneOf(['github', 'start', 'pause', 'restart', 'default']),
```

Comment:
You don't need a `default` state in Icon to render a Button without an icon. In the Button, just don't render the icon if the icon name isn't defined.